src/freenet/clients/fcp/RequestStatus.java

```diff
@@ -136,17 +153,15 @@ public boolean isStarted() {
 	public synchronized void updateStatus(SplitfileProgressEvent event) {
 		this.failedBlocks = event.failedBlocks;
 		this.fatallyFailedBlocks = event.fatallyFailedBlocks;
+    	// clone() because Date is mutable
+		this.latestFailure = event.latestFailure != null ? (Date)event.latestFailure.clone() : null;
 		this.fetchedBlocks = event.succeedBlocks;
+        // clone() because Date is mutable
+		this.latestSuccess = event.latestSuccess != null ? (Date)event.latestSuccess.clone() : null;
```

Comment:
Whitespace fixed by 3418c7daa580672fa0239b41113ac27ec388f2d2.
Done this since I assume that people prefer having 1468 soon to having an answer about whether billing probably less than 5 EUR here is OK.